src/execution/operator/csv_scanner/state_machine/csv_state_machine_cache.cpp

```diff
@@ -163,6 +163,33 @@ void CSVStateMachineCache::Insert(const CSVStateMachineOptions &state_machine_op
 	transition_array.skip_quoted[escape] = false;
 	transition_array.skip_quoted[static_cast<uint8_t>('\n')] = false;
 	transition_array.skip_quoted[static_cast<uint8_t>('\r')] = false;
+
+	transition_array.delimiter = delimiter;
+	transition_array.new_line = static_cast<uint8_t>('\n');
+	transition_array.carriage_return = static_cast<uint8_t>('\r');
+	transition_array.quote = quote;
+	transition_array.escape = escape;
+
+	// Shift and OR to replicate across all bytes
+	transition_array.delimiter |= transition_array.delimiter << 8;
+	transition_array.delimiter |= transition_array.delimiter << 16;
+	transition_array.delimiter |= transition_array.delimiter << 32;
+
+	transition_array.new_line |= transition_array.new_line << 8;
+	transition_array.new_line |= transition_array.new_line << 16;
+	transition_array.new_line |= transition_array.new_line << 32;
+
+	transition_array.carriage_return |= transition_array.carriage_return << 8;
+	transition_array.carriage_return |= transition_array.carriage_return << 16;
+	transition_array.carriage_return |= transition_array.carriage_return << 32;
```

Comment:
This appears to be unused (which is likely why the Windows tests are failing?)

Reply:
Yeah, ive added it before your comment, just forgot to press the ready for review button 😅
https://github.com/duckdb/duckdb/pull/10855/commits/00ff370b7850c5a98ede031f9cebb119dfdb4ab6